test/TestProjects/ResourceClients-LowLevel/readme.md

```diff
@@ -5,7 +5,6 @@
 ``` yaml
 require: $(this-folder)/../../../readme.md
 input-file: $(this-folder)/ResourceClients-LowLevel.json
-data-plane: true
```

Comment:
Could we keep this setting in the manually-edited files?

Reply:
We don't need to maintain data-plane flag anymore. if neither azure-arm nor generation1-convenience-client is set, it will be DPG. No need a flag for DPG.

Reply:
we only have two flag for customer to set. azure-am for management-plane, eneration1-convenience-client is for HLC.